modal/_functions.py

```diff
@@ -652,11 +652,11 @@ def from_local(
             for method_name, partial_function in interface_methods.items():
                 function_type = get_function_type(partial_function.params.is_generator)
                 function_name = f"{info.user_cls.__name__}.{method_name}"
-
-                if config.get("function_schemas") and not partial_function._is_web_endpoint():
-                    method_schema = get_callable_schema(partial_function.raw_f, ignore_first_argument=True)
-                else:
-                    method_schema = None
+                method_schema = get_callable_schema(
+                    partial_function._get_raw_f(),
+                    is_web_endpoint=partial_function._is_web_endpoint(),
+                    ignore_first_argument=True,
+                )
```

Comment:
Could be nice if `get_callable_schema` could just accept the `Function`/`PartialFunction` (or ... `Interface`? Some other correct type?) and then consistently access the underlying callable and metadata like `is_web_endpoint`. Food for thought.